README.md

```diff
@@ -97,6 +97,10 @@
   - [Snappydata](https://github.com/SnappyDataInc/snappydata) - SnappyData: OLTP + OLAP Database built on Apache Spark.
   - [TimescaleDB](https://www.timescale.com/) - Built as an extension on top of PostgreSQL, TimescaleDB is a time-series SQL database providing fast analytics, scalability, with automated data management on a proven storage engine.
 
+## Databricks cluster management
+
+- [Gradient](https://synccomputing.com/) - Automated cluster management for Databricks. Gradient by Sync uses an advanced ML algorithm to contionusly optimize your workspace, saving you 50% on your bill while meeting SLAs. 
+
```

Comment:
I don't think we should add a separate section just for Databricks. Please find a more suitable section. 

Can you update the description to be more focused on the function of Gradient instead of a sales pitch on how it will save time and money.